register-autonomoustuff-repository/action.yaml

```diff
@@ -0,0 +1,18 @@
+name: register-autonomoustuff-repository
+description: register-autonomoustuff-repository
+
+inputs:
+  rosdistro:
+    description: ROS distro
+    required: true
+
+runs:
+  using: composite
+  steps:
+    # https://github.com/astuff/pacmod3_msgs#installation
+    - name: Register Autonomoustuff repository
+      run: |
+        apt-get -y update
+        apt-get -y install apt-transport-https
+        sh -c 'echo "deb [trusted=yes] https://s3.amazonaws.com/autonomoustuff-repo/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/autonomoustuff-public.list'
```

Comment:
@wep21 For what do you need [this](https://github.com/autowarefoundation/autoware-github-actions/pull/6/files#diff-e8bcce185d11a3bd5d4e54e91a38ec991238ef470148394f9d27211bd660945eR19)? I tentatively removed it since there is no description in https://github.com/astuff/pacmod3_msgs.

Reply:
The first line is to register rosdep rules, while the second line is to register apt repository.
https://github.com/astuff/docker-builds/blob/master/ros2/foxy/ros-core/Dockerfile

Reply:
4041df5863a083f1842ae74b1e73cb0827b3465e